### DISCUSSION TITLE
Noob using Photoprism - 2 things not wokring.

### DISCUSSION THREAD

**Discussion by Daniel-PT** [photoprism/photoprism#3936] [Q&A] [answered]
HI all
Im way out and need some help. Just cant thees simple things to work?

Im running Photoprism in Unraid and a test path for now.

1: I have made a test path in photoprism called 
/mnt/user/data/media/pictures/Testphotoprism/
I added 3 picturs and they was showing in PP.
But then i deleted the 3 pictures (with SMB)
But the pictures is still in PP.
So i found that you can run some scripts every hour or someting to "scan" the /originals folder.
I tryed
cd /mnt/user/data/media/pictures/Testphotoprism
docker exec -t PhotoPrism photoprism index

https://forums.unraid.net/topic/131591-photoprism-how-i-use-a-simple-user-script-to-automate-scheduled-sync-without-webdav/#comment-1335194

and
docker exec PhotoPrism photoprism index "/mnt/user/data/media/pictures/Testphotoprism" --cleanup

But nothing happends?

Log:
"time="2023-12-06T19:43:13+01:00" level=debug msg="index: running face recognition"
time="2023-12-06T19:43:13+01:00" level=debug msg="faces: found no orphan markers [1.274095ms]"
time="2023-12-06T19:43:13+01:00" level=debug msg="markers: found no invalid references [314.327µs]"
time="2023-12-06T19:43:13+01:00" level=debug msg="markers: found no missing subjects [123.362µs]"
time="2023-12-06T19:43:13+01:00" level=debug msg="faces: found no ambiguous subjects [102.767µs]"
time="2023-12-06T19:43:13+01:00" level=debug msg="faces: found no clusters to be merged [83.185µs]"
time="2023-12-06T19:43:13+01:00" level=debug msg="faces: found no existing clusters"
time="2023-12-06T19:43:13+01:00" level=debug msg="faces: skipped clustering"
time="2023-12-06T19:43:13+01:00" level=debug msg="faces: found no new faces [261.862µs]"
time="2023-12-06T19:43:13+01:00" level=debug msg="faces: found no unmatched markers"
time="2023-12-06T19:43:13+01:00" level=debug msg="faces: updated 0 markers, recognized 0 faces, 0 unknown [230.689µs]"
time="2023-12-06T19:43:13+01:00" level=debug msg="index: updating metadata"
time="2023-12-06T19:43:13+01:00" level=debug msg="moments: analyzing 3 photos and 0 videos, with threshold 3"
time="2023-12-06T19:43:13+01:00" level=debug msg="index: updating counts"
time="2023-12-06T19:43:13+01:00" level=debug msg="counts: updated 1 place [96.27µs]"
time="2023-12-06T19:43:13+01:00" level=debug msg="counts: updated 0 subjects [174.28µs]"
time="2023-12-06T19:43:13+01:00" level=debug msg="counts: updated 4 labels [205.964µs]"
time="2023-12-06T19:43:13+01:00" level=debug msg="index: updating covers"
time="2023-12-06T19:43:13+01:00" level=debug msg="covers: updated 1 album [1.506129ms]"
time="2023-12-06T19:43:13+01:00" level=debug msg="covers: updated 2 folders [1.537154ms]"
time="2023-12-06T19:43:13+01:00" level=debug msg="covers: updated 2 months [1.544068ms]"
time="2023-12-06T19:43:13+01:00" level=debug msg="covers: updated 5 labels [3.248439ms]"
time="2023-12-06T19:43:13+01:00" level=debug msg="covers: updated 0 subjects [91.315µs]"
time="2023-12-06T19:49:46+01:00" level=debug msg="server: GET /api/v1/config (200) [1.903972ms]"
time="2023-12-06T19:52:16+01:00" level=debug msg="photos: found 3 results for public:true dist:2.000000 camera:0 lens:0 year:0 month:0 quality:3 count:120 order:newest merged:true [899.743µs]""

"root@Kratos:~# cd /mnt/user/data/media/pictures/Testphotoprism
docker exec -t PhotoPrism photoprism index
INFO[2023-12-06T19:42:12+01:00] Become a member today, support our mission and enjoy our member benefits! 💎 
INFO[2023-12-06T19:42:12+01:00] Visit https://www.photoprism.app/membership to learn more. 
DEBU[2023-12-06T19:42:12+01:00] config: successfully initialized [1.396609ms] 
DEBU[2023-12-06T19:42:12+01:00] migrate: running database migrations         
DEBU[2023-12-06T19:42:12+01:00] migrate: 20221015-100000 skipped             
DEBU[2023-12-06T19:42:12+01:00] migrate: 20221015-100100 skipped             
DEBU[2023-12-06T19:42:12+01:00] migrate: 20211121-094727 skipped             
DEBU[2023-12-06T19:42:12+01:00] migrate: 20211124-120008 skipped             
DEBU[2023-12-06T19:42:12+01:00] migrate: 20220329-040000 skipped             
DEBU[2023-12-06T19:42:12+01:00] migrate: 20220329-050000 skipped             
DEBU[2023-12-06T19:42:12+01:00] migrate: 20220329-061000 skipped             
DEBU[2023-12-06T19:42:12+01:00] migrate: 20220329-071000 skipped             
DEBU[2023-12-06T19:42:12+01:00] migrate: 20220329-081000 skipped             
DEBU[2023-12-06T19:42:12+01:00] migrate: 20220329-083000 skipped             
DEBU[2023-12-06T19:42:12+01:00] migrate: 20220329-091000 skipped             
DEBU[2023-12-06T19:42:12+01:00] migrate: 20220329-093000 skipped             
DEBU[2023-12-06T19:42:12+01:00] migrate: 20220421-200000 skipped             
DEBU[2023-12-06T19:42:12+01:00] migrate: 20230309-000001 skipped             
DEBU[2023-12-06T19:42:12+01:00] migrate: 20230313-000001 skipped             
DEBU[2023-12-06T19:42:12+01:00] migrate: completed in 10.37517ms             
INFO[2023-12-06T19:42:12+01:00] indexing originals in /photoprism/originals  
INFO[2023-12-06T19:42:12+01:00] originals folder is empty                    
INFO[2023-12-06T19:42:12+01:00] index: updated 0 files [256.704µs]           
INFO[2023-12-06T19:42:12+01:00] indexed 0 files in 15.411313ms               
DEBU[2023-12-06T19:42:12+01:00] closed database connection                   
root@Kratos:/mnt/user/data/media/pictures/Testphotoprism# "

What am i doing wroung? 

Please help :)

**Comment by graciousgrey** [comment, marked as answer]
This question has already been answered in the chat. I will also add the answers here in case someone has a similar question :)

For Webdav access, it is important that Webdav is enabled for the user you are logging in with. Other reasons for a failed Webdav connection can be found in the following checklist:

https://docs.photoprism.app/getting-started/troubleshooting/#no-webdav-access

If files are deleted in the file system and not via the PhotoPrism user interface, a normal index is required so that PhotoPrism recognizes that the files have been deleted and removes them from the index. 

https://docs.photoprism.app/user-guide/library/originals/

In the special case that the "Originals" folder is completely empty (because all files have been deleted), PhotoPrism does not remove the files from the index. This is a safety mechanism. We have implemented it so that not all files are removed from the index during indexing if, for example, the original folder cannot be accessed due to network problems.

- https://github.com/photoprism/photoprism/issues/2299

**Reply by Daniel-PT** [reply]
1: yes - So i made a new user "Daniel" and logged in with that into the webui. And i saw that webdav was NOT disabled then i didnt thought that was the problem.
Then i used
photoprism users ls
Saw that webdav was empty.
Then
hotoprism users mod --webdav, -w Daniel
and now it works.

2: Ahhh okay that was not clear to me! :) Meny thanks i can see that it works now!

**Reply by lastzero** [reply]
That sounds like webdav was automatically disabled by the system? Can you reproduce this?

**Reply by Daniel-PT** [reply]
Yeap i did just make a user "Daniel"
![image](https://github.com/photoprism/photoprism/assets/20338866/cf538ed4-f1b3-4337-bcf1-38980886ad85)

After photoprism users mod --webdav, -w Daniel
![image](https://github.com/photoprism/photoprism/assets/20338866/23ae56de-eff9-424e-bb71-266b49487907)

**Reply by lastzero** [reply]
If the value is empty, it means disabled or no - depending on the context. We thought it would be easier to read this way, just like a disabled checkbox in the Web UI looks "empty" `[ ]` compared to a selected one `[x]`...
